### PR TITLE
feat(CKEditor): Add missing typings for `focusManager`

### PR DESCRIPTION
**Pull request by adrienverge**
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] adds corresponding tests to the test file.

### CODE REVIEW

File: ckeditor/ckeditor.d.ts [hunk 1]
```diff
@@ -544,8 +544,13 @@ declare namespace CKEDITOR {
     }
 
 
-    interface focusManager {
-
+    class focusManager {
+        focus(element: dom.element): void;
```
Comment (vladima) [inline ckeditor/ckeditor.d.ts]: per [docs](http://docs.ckeditor.com/#!/api/CKEDITOR.focusManager) `element` is named `currentActive` and is optional

**Comment by adrienverge** [comment]
Thanks for detecting the typo @vladima!

I fixed it, and also added constructor and properties to the class.

**Comment by vladima** [comment]
thanks!

**Comment by adrienverge** [comment]
You're welcome!

**Pull request merged**